### PR TITLE
April 2018 update - try matrix now update and many projects added

### PR DESCRIPTION
**Pull request by benparsons**

### CODE REVIEW

File: jekyll/_posts/projects/2018-04-24-olm.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+language: C++
+license: Apache
+---
+
+# {{ page.title }}
+An implementation of the Double Ratchet cryptographic ratchet described by
+https://whispersystems.org/docs/specifications/doubleratchet/, written in C and
+C++11 and exposed as a C API.
+
+The specification of the Olm ratchet can be found in `<docs/olm.rst>`_.
```
Comment (richvdh) [inline jekyll/_posts/projects/2018-04-24-olm.md]: that link looks broken. and isn't valid markdown.

File: .gitignore [hunk 2]
```diff
@@ -16,14 +16,10 @@
...
 /blog/
-
-# should not be committed
-site.css
-docs/
-
-.DS_Store
\ No newline at end of file
+/site.css
+/docs
```
Comment (richvdh) [inline .gitignore]: is it true that this is from /blog?

Comment (benparsons) [reply .gitignore]: Not strictly for site.css. Probably doesn't need a comment at all?

Comment (richvdh) [reply .gitignore]: indeed. I'd prefer no comment over a misleading one.

**Pull request merged**